virttest/qemu_vm.py

```diff
@@ -1694,6 +1695,21 @@ def sort_key(dev):
                                     numa_mem, numa_cpus, numa_nodeid)
             devices.insert(StrDev('numa', cmdline=cmdline))
 
+        # add '-numa dist'
+        if devices.has_option("numa dist,.*"):
+            for numa_node in params.objects("guest_numa_nodes"):
+                numa_params = params.object_params(numa_node)
+                numa_nodeid = numa_params.get("numa_nodeid")
+                numa_dist = ast.literal_eval(numa_params.get("numa_dist", "[]"))
+                if numa_nodeid is None or not numa_dist:
+                    continue
+                for dst_distance_pair in numa_dist:
+                    dst_id = dst_distance_pair[0]
+                    distance = dst_distance_pair[1]
+                    cmd = " -numa dist,src=%s,dst=%s,val=%s" % (numa_nodeid,
+                                                                dst_id, distance)
```

Comment:
Still not short enough, recommend:
```
cmd = " -numa dist,src=%s,dst=%s,val=%s"
cmd = cmd % (numa_nodeid, dst_id, distance)
```

Reply:
I always use 80 as a limit,  is it too long ? Or, limited to 72 ?

Reply:
@yanan-fu This line is 81.

Reply:
Updated, thanks!